metadata.json

```diff
@@ -23,6 +23,10 @@
     {
       "name": "puppetlabs/reboot",
       "version_requirement": ">=2.0.0 < 5.0.0"
+    },
+    {
+      "name": "puppet/archive",
+      "version_requirement": ">= v1.0.0"
```

Comment:
please add an upper limit. As lower limit I usually recommend the current version because nobody tested if v1.0.0 actually works.

Reply:
Solved via e5eab7658787544e3cd3da3d33df5986a023287f
Since I am testing with version 4.4.0 of the archive-module I felt free to use it as lower limit.